app/src/main/java/com/google/samples/apps/sunflower/PlantListFragment.kt

```diff
@@ -69,9 +86,20 @@ class PlantListFragment : Fragment() {
         }
     }
 
-    private fun subscribeUi(adapter: PlantAdapter) {
+    private fun subscribeUi(
+        databinding: FragmentPlantListBinding,
+        adapter: PlantAdapter,
+        recyclerViewState: Parcelable?
+    ) {
         viewModel.getPlants().observe(this, Observer { plants ->
-            if (plants != null) adapter.values = plants
+            if (plants != null) {
+                adapter.values = plants
+                databinding.loadingUi.visibility = GONE
+
```

Comment:
If I were you, I would omit static import and change a line above to `databinding.loadingUi.visibility = View.GONE`

Reply:
Ya. I will change.